examples/mqtt-service-ssl/mqtt_service_ssl.bal

```diff
@@ -0,0 +1,28 @@
+import ballerina/lang.value;
+import ballerina/log;
+import ballerina/mqtt;
+import ballerina/time;
+import ballerina/uuid;
+
+public type TemperatureDetails readonly & record {
+    string deviceId;
+    time:Utc timestamp;
+    decimal temperature;
+};
+
+listener mqtt:Listener temperatureSubscriber = new ("ssl://localhost:8883", uuid:createType1AsString(), "mqtt/topic", {
+    connectionConfig: {
+        // Provide the relevant secure socket configurations by using `mqtt:SecureSocket`.
+        secureSocket: {
+            cert: "./resources/path/to/public.crt"
```

Comment:
```suggestion
            cert: "./resource/path/to/public.crt"
```

Reply:
Didnt change as wanted to show that these are in resources folder